playbook/buildfarmer/README.md

```diff
@@ -1,28 +1,45 @@
 # What is the buildfarmer role?
 
-As stated in the [onboarding guide](../../docs/onboarding.md):
-
-> The buildfarmer is in charge of monitoring build regressions, [test regresions](./glossary.md#test-regression), [flaky tests](./glossary.md#flaky-tests) and general warnings on the buildfarm. As well, it reports any issue to the corresponding project team and tracks the status until it's resolved.
+> The buildfarmer is in charge of monitoring [build regressions](#build-regression), [test regresions](#test-regression), [flaky tests](#flaky-test) and general warnings on the buildfarm. As well, it reports any issue to the corresponding project team and tracks the status until it's resolved.
 
 # How does the buildfarmer role work?
 
-Every buildfarmer should follow the [priorities document](../../docs/priorities.md) for triaging issues.
-
-TL;DR:
+Every buildfarmer should follow the priorities defined for triaging issues.
 
-1. Keep the buildfarm up and running
-2. Keep enough machines
-3. Check build regressions
+1. Keep the buildfarms up and running (build.ros2.org, build.osrfoundation.org, ci.ros2.org, etc.)
+2. Keep enough machines to handle current demand for CI
+3. Check build regressions 
 4. Check test regressions
 
 For items 1 and 2 refer to the [priorities document](../../docs/priorities.md#priorities) items 1 and 2.
 
 Items 3 and 4 are covered in [Looking for regressions](./looking_for_regressions.md).
 
+> **Note**
+> For ROS, we prioritize regressions based on [REP2000 support tiers](https://github.com/ros-infrastructure/rep/blob/master/rep-2000.rst#support-tiers) for each distribution
+> E.g., we will check regressions in linux builds before checking regressions in RHEL builds.
+> 
+> In the case of Gazebo, all packages and distributions are the same priority.
```

Comment:
This statement is misleading, I would say something along the lines of `all the packages in each official distribution for each supported OS`